singlecellmultiomics/bamProcessing/bamToCountTable.py

```diff
@@ -615,6 +620,11 @@ def create_count_table(args, return_df=False):
         '-head',
         type=int,
         help='Run the algorithm only on the first N reads to check if the result looks like what you expect.')
+    argparser.add_argument(
+        '-bulk',
```

Comment:
Could you change this too `--bulk` ? The (no-parameter) flags have a double dash in scmo.

Reply:
Thanks, Done! I also changed the type of argument, to an action? type like the rest.

Reply:
Ah yes, I did not see that you did not use the `action='store_true'`